### PR TITLE
[BTCV2-1425] Fix regression test cases

### PR DESCRIPTION
**Pull request by dgabka**
## What does this PR do?

This PR fixes all not skipped regression test cases. All of them should work correctly both on iOS and Android.

## Todos:

- [x] Checked on iOS
- [x] Checked on Android

### CODE REVIEW

File: tests/e2e/specFiles/addingWallets/general.spec.ts [hunk 1]
```diff
@@ -1,29 +1,37 @@
...
 
 import { expectToBeDisabled } from '../../assertions';
+import { isBeta } from '../../helpers/utils';
 import app from '../../pageObjects';
 import steps from '../../steps';
 
 describe('Adding wallet', () => {
   describe('General', () => {
+    beforeEach(async () => {
+      isBeta() && (await app.onboarding.betaVersionScreen.close());
```
Comment (PanRadaCloudbest) [inline tests/e2e/specFiles/addingWallets/general.spec.ts:11]: we still test beta version?

File: tests/e2e/specFiles/addressBook.spec.ts [hunk 2]
```diff
@@ -61,15 +61,15 @@ describe('Address book', () => {
...
         await waitFor(app.addressBook.newContact.addNewContactScreen.nameValidationError)
           .toBeVisible()
           .withTimeout(WAIT_FOR_ELEMENT_TIMEOUT.DEFAULT);
       });
 
       it("shouldn't be possible to create a new contact if address is invalid", async () => {
         await app.addressBook.contactsScreen.tapOnCreateButton();
+        await app.addressBook.newContact.addNewContactScreen.typeName(`Heisenberg`);
         await app.addressBook.newContact.addNewContactScreen.typeAddress('fOoBa5bAZ');
         await app.addressBook.newContact.addNewContactScreen.submit(); // TODO: Remove it once it's fixed in the app
```
Comment (PanRadaCloudbest) [inline tests/e2e/specFiles/addressBook.spec.ts:74]: this comment we need still keep ?

File: tests/e2e/specFiles/addingWallets/general.spec.ts (hunk 1 above)
Comment (aleparty) [reply tests/e2e/specFiles/addingWallets/general.spec.ts:11]: IMO we have still Beta version in environments, if we want to get rid of that we should clean up the app as well

File: tests/e2e/specFiles/addressBook.spec.ts (hunk 2 above)
Comment (dgabka) [reply tests/e2e/specFiles/addressBook.spec.ts:74]: As discussed on the daily, I am going to resolve todos later. However in this particular case the test case works as it is, I don't know what's the fix supposed to be - there is no reference to any Jira ticket. I guess it's about the address format being validated by the form, before submission, but that's just a guess.

File: tests/e2e/specFiles/addingWallets/general.spec.ts (hunk 1 above)
Comment (dgabka) [reply tests/e2e/specFiles/addingWallets/general.spec.ts:11]: Discussed on daily - beta version is still possible to build so tests should be able to handle it.

**Review by PanRadaCloudbest — APPROVED** [review]

File: tests/e2e/specFiles/settings.spec.ts [hunk 3]
```diff
@@ -119,7 +119,7 @@ describe('Settings', () => {
   });
 
   describe('Email notifications', () => {
-    describe('Add email', () => {
+    describe.skip('Add email', () => {
```
[the remark this answers is not in the document]
Comment (dgabka) [reply tests/e2e/specFiles/settings.spec.ts:122]: Email notification handling is currently broken in the tests. There is a separate ticket for it - BTCV2-1337. All tests requiring email operations must be skipped until it's done.

**Pull request merged**